t/plugin/openid-connect.t

```diff
@@ -1487,3 +1487,140 @@ GET /t
 {"access_token_in_authorization_header":false,"bearer_only":false,"client_id":"kbyuFDidLLm280LIwVFiazOqjO3ty8KH","client_secret":"60Op4HFM0I8ajz0WdiStAbziZ-VFQttXuxixHHs2R7r7-CW8GR79l-mmLqMhc-Sa","discovery":"http://127.0.0.1:1980/.well-known/openid-configuration","introspection_endpoint_auth_method":"client_secret_basic","logout_path":"/logout","realm":"apisix","scope":"openid","set_access_token_header":true,"set_id_token_header":true,"set_userinfo_header":true,"ssl_verify":false,"timeout":3}
 --- no_error_log
 [error]
+
+
+
+=== TEST 25: Update plugin with ID provider jwks endpoint for token verification.
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{ "plugins": {
+                            "openid-connect": {
+                                "client_id": "kbyuFDidLLm280LIwVFiazOqjO3ty8KH",
+                                "client_secret": "60Op4HFM0I8ajz0WdiStAbziZ-VFQttXuxixHHs2R7r7-CW8GR79l-mmLqMhc-Sa",
+                                "discovery": "https://samples.auth0.com/.well-known/openid-configuration",
```

Comment:
@spacewander  Is it ok to rely on external services in a test case?  I'm a bit worried. Although I have found that some of our test cases rely on external services like httpbin.org.

Reply:
@fbartels  Will `https://samples.auth0.com/.well-known/openid-configuration`  be valid for a long time (at least 2 years)?

Reply:
@tzssangglass that is a good question. I could not really find much information about this domain in the Auth0 documentation.

I mostly copied the tests based on the public_key tests that were already in the testsuite. Other tests besides them were also using Auth0.

Reply:
I get, my mistake😓, let's keep this.

Reply:
No problem at all